MetaMask/metamask-extension
# fix: Blockaid false positive report

## **Description**
This PR fixes addresses a couple of issues, 

1. The first is that whenever we are triggering a malicious transaction, in the case there are no details (i.e. at the moment, Send ETH) we don't have the possibility to report a false positive. We fixed this by showing More as the accordion title
2. The second issue is that if a user wants to report a false positive for Blockaid, the user is redirected to customer service and needs to search and provide all the information needed manually. We have improved this flow by replacing the link with a new one that leads to a form and pass some data directly to this link to pre-fill the form for the user. 
3. We have also replaced the contact us text with Report and issue/problem text

## **Related issues**

Fixes:
1. [#1696](https://github.com/MetaMask/MetaMask-planning/issues/1696)
2. #21188 

## **Manual testing steps**

1. Build and launch Metamask on main
2. Open testdapp
3. Click on ppom smalicious end eth button
4. Observe you see the blockaid banner as shown in the first screen in before section
5. Checkout this branch, build and launch MM
6. Repeat 2,3
7. Observe the first and second screenshots in after section
8. Click on ppom malicios signature button
9. Observe the third and fourth screenshots in after section 

## **Screenshots/Recordings**

### **Before**

<img width="365" alt="Screenshot 2023-12-13 at 21 32 18" src="https://github.com/MetaMask/metamask-extension/assets/44811/259f46e6-0975-49ae-b180-5d87be244b05">

### **After**
<img width="369" alt="Screenshot 2023-12-13 at 21 10 13" src="https://github.com/MetaMask/metamask-extension/assets/44811/719bcff2-4d14-4dde-acfa-9717dfaf9c98">

<img width="359" alt="Screenshot 2023-12-19 at 17 51 16" src="https://github.com/MetaMask/metamask-extension/assets/44811/524ea5a9-2559-4c35-93df-e0c85e8b72e4">


<img width="367" alt="Screenshot 2023-12-13 at 21 11 02" src="https://github.com/MetaMask/metamask-extension/assets/44811/2d0142ed-be8a-494d-9283-090396563cb6">


<img width="371" alt="Screenshot 2023-12-19 at 17 51 50" src="https://github.com/MetaMask/metamask-extension/assets/44811/91e80954-2933-48e6-8e81-b91814e2b1a1">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
